lib/ibm/cloud/sdk/vpc/instances.rb

```diff
@@ -38,21 +38,115 @@ def params(start: nil, limit: nil, name: nil, vpc_id: nil, vpc_crn: nil, vpc_nam
 
         # Work with a single instance.
         class Instance < VPCInstance
+          TRANSITIONAL_STATES = %(pausing pending restarting resuming starting stopping)
+          ERROR_STATE = 'failed'
+          RUNNING_STATE = 'running'
+          STOPPED_STATES = %('stopped', 'paused')
```

Comment:
Same here, should be `%w[stopped paused]`

Reply:
Good catch, thanks